pkg/costmodel/costmodel.go

```diff
@@ -1136,7 +1136,7 @@ func (cm *CostModel) GetNodeCost(cp costAnalyzerCloud.Provider) (map[string]*cos
 
 		cnode, err := cp.NodePricing(cp.GetKey(nodeLabels, n))
 		if err != nil {
-			log.DedupedWarningf(10, "Error getting node pricing. Error: %s", err.Error())
+			klog.Infof("Error getting node pricing. Error: %s", err.Error())
```

Comment:
Was this for testing, or do we want to un-dedupe this log in production?

Reply:
I think we will want to undedupe this in prod, unless there are major concerns. node pricing issues have been our top support questions so far.